Converter/igtlioNDArrayConverter.h

```diff
@@ -0,0 +1,27 @@
+#ifndef NDARRAYCONVERTER_H
+#define NDARRAYCONVERTER_H
+
+#include "igtlioConverterExport.h"
+#include <igtlNDArrayMessage.h>
+#include "igtlioBaseConverter.h"
+class vtkDataArray;
+typedef vtkSmartPointer<class igtlioNDArrayDevice> igtlioNDArrayDevicePointer;
+class OPENIGTLINKIO_CONVERTER_EXPORT igtlioNDArrayConverter : public igtlioBaseConverter
+{
+public:
+
+
+    struct ContentData
+    {
+    vtkSmartPointer<vtkDataArray> NDArray_msg;
+    };
```

Comment:
The ContentData will also have to store what kind of scalar the array is. This could re-use the vtk type define macros if you want